test/func/docker/Dockerfile

```diff
@@ -1,15 +1,26 @@
-FROM cimg/node:16.20-browsers
+FROM node:18
```

Comment:
I see that the resulting images size is around 1GB. Have you tried making it smaller? One obvious thing that comes to mind is that `-slim` version of node:18 image weights only 60MB vs 400MB for the default one: https://hub.docker.com/_/node/tags?page=&page_size=&ordering=&name=18